src/qt/frame.cpp

```diff
@@ -49,8 +49,11 @@ class wxQtCentralWidget : public wxQtEventSignalHandler< QWidget, wxFrame >
 
 wxFrame::~wxFrame()
 {
-    // central widget should be deleted by qt when the main window is destroyed
-    QtStoreWindowPointer( GetQMainWindow()->centralWidget(), nullptr );
+    if ( GetQMainWindow() && GetQMainWindow()->centralWidget() )
```

Comment:
I think it could be useful to mention that this test is false for MDI children here too, otherwise it's not very clear.